### PR TITLE
Allow user to specify which tables to use at command line.

### PR DESCRIPTION
**Pull request by lajones**
Further fixes for issue #2751. Updates the command-line and powershell command to allow the user to make use of the framework set up in PR #3214.

### CODE REVIEW

File: src/EntityFramework.Relational.Design/ReverseEngineering/Internal/TableSelectionSetBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+{
+    // TODO:
+    // Does not support schema or table names containing FilterSeparator.
+    // Does not support schema names (or schemaless table names) that start with ExcludeIndicator.
+    // Does not support schema names that contain SchemaTableSeparator.
+    public static class TableSelectionSetBuilder
+    {
+        private static readonly char[] FilterSeparator = new [] { ',' };
+        private static readonly string ExcludeIndicator = "-";
+        private static readonly char SchemaTableSeparator = ':';
```
Comment (bricelam) [inline src/EntityFramework.Relational.Design/ReverseEngineering/Internal/TableSelectionSetBuilder.cs]: Again, let's discuss this format at a design meeting.

**Comment by bricelam** [comment]
:shipit:

File: src/EntityFramework.Relational.Design/ReverseEngineering/Internal/TableSelectionSetBuilder.cs (hunk 1 above)
Comment (lajones) [inline src/EntityFramework.Relational.Design/ReverseEngineering/Internal/TableSelectionSetBuilder.cs]: Yep. Sounds good.

**Comment by lajones** [comment]
Thanks. Checked in with fcfe8ef.

**Pull request closed without merging**